cloudformation/xtdb_alb.yml

```diff
@@ -0,0 +1,79 @@
+AWSTemplateFormatVersion: '2010-09-09'
+
+Parameters:
+  VpcId: 
+    Description: ID of the VPC to host the LoadBalancer on
+    Type: AWS::EC2::VPC::Id
+
+  SecurityGroupId:
+    Description: Group ID of the security group to be used by the ECS nodes
+    Type: AWS::EC2::SecurityGroup::Id
+
+  PublicSubnets:
+    Description: List of public subnets to host the load balancer on
+    Type: List<AWS::EC2::Subnet::Id>
+
+Resources:
+  ECSALB:
+    Type: 'AWS::ElasticLoadBalancingV2::LoadBalancer'
+    Properties:
+      Name: ECSALB
+      Scheme: internet-facing
+      LoadBalancerAttributes:
+        - Key: idle_timeout.timeout_seconds
+          Value: '30'
+      Subnets: !Ref PublicSubnets
+      SecurityGroups:
+        - !Ref SecurityGroupId
+
+  ECSTG:
+    Type: 'AWS::ElasticLoadBalancingV2::TargetGroup'
+    DependsOn: ECSALB
+    Properties:
+      HealthCheckIntervalSeconds: 10
+      HealthCheckPath: /status
+      HealthCheckProtocol: HTTP
+      HealthCheckTimeoutSeconds: 5
+      HealthyThresholdCount: 2
+      Name: ECSTG
+      Port: 3000
+      Protocol: HTTP
+      UnhealthyThresholdCount: 2
+      VpcId: !Ref VpcId
+
+  ALBListener:
+    Type: 'AWS::ElasticLoadBalancingV2::Listener'
+    Properties:
+      DefaultActions:
+        - Type: forward
+          TargetGroupArn: !Ref ECSTG
+      LoadBalancerArn: !Ref ECSALB
+      Port: '3000'
```

Comment:
this the external port? if so, 443?

Reply:
As discussed - port 80 for now, will add instructions for setting up HTTPs

Steps would be, in essence:
- Set up SSL certificate in ACM
- Update vpc security group to allow traffic on port 443
- Update ALB exposed port to 443, protocol to HTTPs, add certificate